daemon/cmd/kube_proxy_replacement_test.go

```diff
@@ -114,25 +114,6 @@ func (s *KPRSuite) TestInitKubeProxyReplacementOptions(c *C) {
 			},
 		},
 
-		// KPR disabled: all options disabled except host legacy routing.
-		{
-			"kpr-disabled",
-			func(cfg *kprConfig) {
-				cfg.kubeProxyReplacement = option.KubeProxyReplacementDisabled
-			},
-			kprConfig{
-				enableSocketLB:          false,
-				enableNodePort:          false,
-				enableHostPort:          false,
-				enableExternalIPs:       false,
-				enableSessionAffinity:   false,
-				enableIPSec:             false,
-				enableHostLegacyRouting: true,
-				enableSocketLBTracing:   false,
-				expectedErrorRegex:      "",
-			},
-		},
-
```

Comment:
Why not testing this with `false` instead?

Reply:
+1

Reply:
good point. I have adjust this test and move it close to other KPR=false test.